odl/solvers/functional/default_functionals.py

```diff
@@ -2193,6 +2198,169 @@ def __repr__(self):
                                          self.__norm.pwisenorm.exponent)
 
 
+class IndicatorSimplex(Functional):
+
+    r"""The indicator functional of a simplex in any dimension.
```

Comment:
I'm not sure about the style we follow, but I think we skip the "The" here, right? Also "in any dimension" is basically implicit in ODL, we usually try to only mention if there is a restriction :)